pkg/gce-pd-csi-driver/node.go

```diff
@@ -340,6 +340,14 @@ func (ns *GCENodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStage
 				devicePath, stagingTargetPath, fstype, options, err))
 	}
 
+	// Part 4: Resize filesystem.
+	resizer := resizefs.NewResizeFs(ns.Mounter)
+	_, err = resizer.Resize(devicePath, stagingTargetPath)
```

Comment:
In Windows this is https://github.com/kubernetes-csi/csi-proxy/blob/202a4e9cfa2c6277920c95f5325c9568bc520531/pkg/os/volume/api.go#L153 with `size=0` which will resize the volume to occupy the entire disk, I think this should work in Windows too.

Do you know if we have a test for this scenario (restore from a snapshot to a larger volume size) in the storage e2e tests in k8s? I checked in https://testgrid.k8s.io/provider-gcp-compute-persistent-disk-csi-driver#ci-windows-2019-provider-gcp-compute-persistent-disk-csi-driver that we're not testing resizing in Windows.

Reply:
Yes, I have this [PR](https://github.com/kubernetes/kubernetes/pull/110304) open which is currently waiting for this PR to merge because the test would fail in CI for (any) driver that does not resize properly. To allow drivers to opt out from the test we've added the[ anti-capability](https://github.com/kubernetes/kubernetes/pull/110304#issuecomment-1156659326).

Reply:
Sounds good, I added a comment in that other PR, in the worst case we might need to add a skip for Windows or find a way to run the `stat` command in Windows too.

Reply:
Awesome, thank you. Are we good for approval now @mattcary?